### PR TITLE
Improve hand reorientation task

### PR DESCRIPTION
**Pull request by thowell**
The hand reorientation task is improved:

- A nominal grasp configuration cost is added
- A hand joint velocity cost is added
- Default interpolation is changed to zero-order hold (from cubic)

Qualitatively the behavior is less jittery and the hand stops moving once a desired cube orientation is achieved.

### CODE REVIEW

File: mjpc/tasks/hand/task.xml [hunk 1]
```diff
@@ -37,13 +37,19 @@
...
+    <user name="Joint Vel." dim="26" user="0 1.0e-4 0.0 1.0e-1" />
     <framepos name="palm_position" objtype="site" objname="grasp_site"/>
     <framequat name="cube_goal_orientation" objtype="body" objname="goal"/>
   </sensor>
 
   <include file="cube.xml"/>
   <include file="right_hand.xml"/>
+
+  <keyframe>
+    <key qpos='1 0 0 0 0.33326 -0.00362331 0.0375343 0.707635 0.70405 0.0500937 -0.0325089 5.55212e-10 -0.235248 -0.178041 0.480484 0.730515 0.6284 -0.059347 0.535468 0.746225 0.56556 -0.03491 0.544632 0.53414 0.793355 0.384846 -0.254843 0.178072 0.761935 0.746225 -0.90042 0.06721 0.01047 0.6981 0.4255'/>
```
Comment (yuvaltassa) [inline mjpc/tasks/hand/task.xml:53]: name="grasp" ?

File: mjpc/tasks/hand/task.xml [hunk 2]
```diff
@@ -8,10 +8,10 @@
     <numeric name="agent_horizon" data="0.25" />
     <numeric name="agent_timestep" data="0.01" />
     <numeric name="agent_policy_width" data="0.0035" />
-    <numeric name="sampling_spline_points" data="6" />
+    <numeric name="sampling_spline_points" data="5" />
```
Comment (yuvaltassa) [inline mjpc/tasks/hand/task.xml:11]: Did you try smaller values?

File: mjpc/tasks/hand/task.xml (hunk 1 above)
Comment (thowell) [reply mjpc/tasks/hand/task.xml:53]: Done.

File: mjpc/tasks/hand/task.xml (hunk 2 above)
Comment (thowell) [reply mjpc/tasks/hand/task.xml:11]: Yes. It works with 2, 3, and 4 as well.

**Comment by yuvaltassa** [comment]
Sorry AFK, can't approve PR from phone, LGTM.

**Review by erez-tom — APPROVED** [review]

**Pull request merged**